src/pkg/caendr/caendr/services/validate.py

```diff
@@ -25,13 +25,13 @@ def get_delimiter_from_filepath(filepath=None, valid_file_extensions=None):
 
 
 
-def validate_file(local_path, columns, delimiter='\t', unique_rows=False):
+def validate_file(local_path_or_file, columns, delimiter='\t', unique_rows=False):
```

Comment:
Generalized to work on `os.PathLike`, so we can pass a `LocalFile` object directly